src/Profile/DeviceConfig.cpp

```diff
@@ -100,7 +100,20 @@ LoadPath(const ProfileMap &map, DeviceConfig &config, unsigned n)
 {
   char buffer[64];
   MakeDeviceSettingName(buffer, "Port", n, "Path");
+#ifdef _WIN32 
+  // on Windows don't use the form 'COM1:' anymore, use 'COM1'
```

Comment:
That's wrong. A different part of this PR changes this aspect, but making this change here breaks the device configuration dialog.

Reply:
Why? I only strip the colon from the profile setting, if the setting is with the old style. So in the dialog, in a new setting, in the opening of port  - overall I use the colonless portname. I don't see a break in the configuration dialog?

Reply:
This code makes the profile settings inconsistent. Some parts of XCSoar still use a trailing colon, others do not. If you believe there should not be a colon, one commit should remove it everywhere, the commit should not do anything else, and the commit message should explain why this change should be done.
Also note that the colon change breaks all downgrades to an older XCSoar version. Such a compatibility breakage should be well justified. I don't see any justification at all.

Reply:
And what is your proposal? To stay with the trailing colon itself for a downgradable setting? Otherwise by stripping the colon an older XCSoar has no connection to this port... But if a user set this to a port COM15 for example he also loses connection to that port. 
I prefer a cut at this point: From now on there should only be settings without a trailing colon - the workaround still takes the old setting into account. I'm not aware that there are other port names in Windows that have a colon after them.
But as you said before: Probably nobody uses Windows in a flight configuration.

Reply:
At this point, I don't propose anything - you have proposed merging your code, but without explaining it. Once I have your explanation, I can review it, but I don't have it. I was just pointing out aspects that should be explained, that you should consider when writing the explanation. Your work is incomplete, so it's your turn.